.github/workflows/tuist.yml

```diff
@@ -83,7 +83,8 @@ jobs:
             'build',
             'cache-frameworks',
             'cache-xcframeworks',
-            'dependencies',
+            'dependencies-fetch',
```

Comment:
do we really want to split these two? I believe it might make sense to keep `dependencies` feature tests at the same place - at least until they grow too large

Reply:
I'm not sure how to combine two features into single `*.feature` file. I have split `tuist dependencies fetch` and `tuist dependencies update` like `tuist lint project` and `tuist lint code` are split. Maybe `dependencies` case is more connected. Would you hint me how the test should look like?   

I have been thinking about:

**version 1**
```ruby
Feature: Install dependencies Tuist.

    Scenario: The project is an application with framework and tests and Dependencies.swift (app_with_framework_and_tests_and_dependencies)
        Given that tuist is available
        And I have a working directory
        Then I copy the fixture app_with_framework_and_tests_and_dependencies into the working directory
        Then tuist fetches dependencies
        Then tuist updates dependencies
        Then a directory Tuist/Dependencies/Carthage/Alamofire.xcframework exists
        Then a file Tuist/Dependencies/Carthage/.Alamofire.version exists
        Then a file Tuist/Dependencies/Lockfiles/Cartfile.resolved exists
        Then a directory Tuist/Dependencies/SwiftPackageManager/checkouts/SnapKit exists
        Then a file Tuist/Dependencies/Lockfiles/Package.resolved exists
```

**version 2**
```ruby
Feature: Install dependencies Tuist.

    Scenario: The project is an application with framework and tests and Dependencies.swift (app_with_framework_and_tests_and_dependencies)
        Given that tuist is available
        And I have a working directory
        Then I copy the fixture app_with_framework_and_tests_and_dependencies into the working directory
        Then tuist fetches dependencies
        Then a directory Tuist/Dependencies/Carthage/Alamofire.xcframework exists
        Then a file Tuist/Dependencies/Carthage/.Alamofire.version exists
        Then a file Tuist/Dependencies/Lockfiles/Cartfile.resolved exists
        Then a directory Tuist/Dependencies/SwiftPackageManager/checkouts/SnapKit exists
        Then a file Tuist/Dependencies/Lockfiles/Package.resolved exists
        Then tuist updates dependencies
        Then a directory Tuist/Dependencies/Carthage/Alamofire.xcframework exists
        Then a file Tuist/Dependencies/Carthage/.Alamofire.version exists
        Then a file Tuist/Dependencies/Lockfiles/Cartfile.resolved exists
        Then a directory Tuist/Dependencies/SwiftPackageManager/checkouts/SnapKit exists
        Then a file Tuist/Dependencies/Lockfiles/Package.resolved exists
```

**version 3**

```ruby
Feature: Install dependencies Tuist.

    Scenario: The project is an application with framework and tests and Dependencies.swift (app_with_framework_and_tests_and_dependencies)
        Given that tuist is available
        And I have a working directory
        Then I copy the fixture app_with_framework_and_tests_and_dependencies into the working directory
        Then tuist fetches dependencies
        Then a directory Tuist/Dependencies/Carthage/Alamofire.xcframework exists
        Then a file Tuist/Dependencies/Carthage/.Alamofire.version exists
        Then a file Tuist/Dependencies/Lockfiles/Cartfile.resolved exists
        Then a directory Tuist/Dependencies/SwiftPackageManager/checkouts/SnapKit exists
        Then a file Tuist/Dependencies/Lockfiles/Package.resolved exists
        # add step that edits `Dependencies.swift`
        Then tuist updates dependencies
        Then a directory Tuist/Dependencies/Carthage/Alamofire.xcframework exists
        Then a file Tuist/Dependencies/Carthage/.Alamofire.version exists
        Then a file Tuist/Dependencies/Lockfiles/Cartfile.resolved exists
        Then a directory Tuist/Dependencies/SwiftPackageManager/checkouts/SnapKit exists
        Then a file Tuist/Dependencies/Lockfiles/Package.resolved exists
```

Reply:
right, let's keep the features separated, it's fine, too, rather than combining `update` and `fetch` into a single scenario